tests/unit/doc/example_with_meiga.py

```diff
@@ -3,12 +3,10 @@
 from meiga import Error, Failure, Result, Success
 
 
-class NoSuchKey(Error):
-    ...
+class NoSuchKey(Error): ...
```

Comment:
I guess this depends on some ruff configuration. 🤔

Reply:
New versions of ruff handle things differently to older versions.